.github/workflows/main.yml

```diff
@@ -88,11 +89,12 @@ jobs:
       - name: Install setuptools
         run: python -m pip install --upgrade setuptools wheel twine
 
-      - name: Release
-        env:
-          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-#          PYPI_TOKEN: ${{ secrets.PYPI_TOKEN }}
-          PYPI_USERNAME: techlee
-          PYPI_PASSWORD: pTL4bF82LZ+Usn7rhm/7tPDz4QiF+RWuVyu+7aNtgqy4JZ5nCloIxohSxODCr58vYabAMecxwCsvCVm7DSWeJiYzFiyWMiEIRxmuc6LHSNnG8pYIHuj9qBcOdyshQbN5ALpuvnpFX2sUQ20hy3ge6cdFT4xLZAm4nzmNnR6e20MNOT675agdNfgDsjnNEBV7lCGfvpYtAJgaB95f8P24Q/WX6JevcrhCLLoLOnZdjt8RxvhV95QF03E3dI9c7d1NwFEZ/IbK2WfXjZ8rY2vYutscSlvag2r+N/ZVlOwTQvwWeZHXkDu6bRsV2mKC1Sq2J6q6mwtFm81YLB5A0RgMIJu+4Ec36Xl1HxRCBgDxDh4WfZkx/yZJsAMqTKVnFsImDt76MD6sDyemG/cPhli3qkAYazgdL2rv9l8KQWVwIhUTN4zOigzDpoEGIvk3DjrAoCoyP/9U5oagBnHSdL/e5jHFoJQ9CS8RRUYvsez6QWE7V06qRLfTSpxmxzvFrA0RENw6RcpabVmbFac/HdMyv9aFL8Z/bWHf2vkyAYUDBDiyFKKYcGWxTsS3L+fTDvyycfy8NJ3jSgloJiXH2FFsEJ03ZxgCIONNMw8gZ58gglJC0jJHZbzjIAl7oNcm+6S0T56y3H6Hb2zQk0eRD6qiY6RQZowc+HH9Is0ejNhW9zc=
+      - name: Python Semantic Release
+        uses: relekang/python-semantic-release@master
+        with:
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+          pypi_token: ${{ secrets.PYPI_TOKEN }}
+#          PYPI_USERNAME: techlee
+#          PYPI_PASSWORD: pTL4bF82LZ+Usn7rhm/7tPDz4QiF+RWuVyu+7aNtgqy4JZ5nCloIxohSxODCr58vYabAMecxwCsvCVm7DSWeJiYzFiyWMiEIRxmuc6LHSNnG8pYIHuj9qBcOdyshQbN5ALpuvnpFX2sUQ20hy3ge6cdFT4xLZAm4nzmNnR6e20MNOT675agdNfgDsjnNEBV7lCGfvpYtAJgaB95f8P24Q/WX6JevcrhCLLoLOnZdjt8RxvhV95QF03E3dI9c7d1NwFEZ/IbK2WfXjZ8rY2vYutscSlvag2r+N/ZVlOwTQvwWeZHXkDu6bRsV2mKC1Sq2J6q6mwtFm81YLB5A0RgMIJu+4Ec36Xl1HxRCBgDxDh4WfZkx/yZJsAMqTKVnFsImDt76MD6sDyemG/cPhli3qkAYazgdL2rv9l8KQWVwIhUTN4zOigzDpoEGIvk3DjrAoCoyP/9U5oagBnHSdL/e5jHFoJQ9CS8RRUYvsez6QWE7V06qRLfTSpxmxzvFrA0RENw6RcpabVmbFac/HdMyv9aFL8Z/bWHf2vkyAYUDBDiyFKKYcGWxTsS3L+fTDvyycfy8NJ3jSgloJiXH2FFsEJ03ZxgCIONNMw8gZ58gglJC0jJHZbzjIAl7oNcm+6S0T56y3H6Hb2zQk0eRD6qiY6RQZowc+HH9Is0ejNhW9zc=
```

Comment:
We use GitHub secrets to store key, not plain text here.

Reply:
Sure, I just commented it out here cause I met some problems. Do we have pypi_token set up in the secrets?

Reply:
@techoner please? Do we have pypi_token implemented or only user and password?

Reply:
@ffyuanda use this:

![image](https://user-images.githubusercontent.com/3787410/122151414-1ce13380-ce92-11eb-91cc-f488ebd5be57.png)

Reply:
@hsluoyz it says 
![image](https://user-images.githubusercontent.com/46557895/122153665-04731800-ce96-11eb-881d-1c19da836793.png)
so I don't think I can access it on my forked repo during a PR process, might need to merge this PR first and see if there are any issues...

Reply:
@ffyuanda that's correct.